README.md

```diff
@@ -12,4 +13,4 @@ information.
 
 ### Subsystem maintainers
 
-* GHCJS - [Matchwood](https://github.com/matchwood)
\ No newline at end of file
+* GHCJS - [Matchwood](https://github.com/matchwood)
```

Comment:
Think this may be a newline issue — can you fix, please?

Reply:
Sure thing!